tests/test_diaCalculationPlugins.py

```diff
@@ -904,6 +912,152 @@ def testCalculate(self):
                                np.sqrt(1 / (n_sources - 1)))
 
 
+class TestMultiLombScarglePeriodogram(unittest.TestCase):
+
+    def GeneratePeriodicData(self, n=10, period=10):
+        """Generate periodic data for testing Lomb Scargle Periodogram.
+
+        Parameters
+        ----------
+        n : int
+            Number of data points.
+        period : float
+            Period of the periodic signal.
+
+        Returns
+        -------
+        t : np.ndarray
+            Time values.
+        y_obs : np.ndarray
+            Observed flux values.
+        """
+        np.random.seed(42)
+
+        t = np.random.randint(100, size=n) + 0.3 + 0.4 * np.random.random(n)
```

Comment:
Any particular reason why you picked these specific values for this random array?

Reply:
Better motivated the time array with appropriate comments.